requirements/dev-requirements.txt

```diff
@@ -10,6 +10,8 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
```

Comment:
@AmitPhulera unfortunately this breaks the build as it's specific to mac os users. I generally manually remove it

Reply:
Ohh!! I did not know that. Thanks for the info. I will remove it.